src/client.rs

```diff
@@ -614,6 +740,19 @@ pub type UnsubscribeConfigurationResponse = dapr_v1::UnsubscribeConfigurationRes
 /// A tonic based gRPC client
 pub type TonicClient = dapr_v1::dapr_client::DaprClient<TonicChannel>;
 
+/// Encryption gRPC request
+type EncryptRequest = crate::dapr::dapr::proto::runtime::v1::EncryptRequest;
+
+/// Decrypt gRPC request
+type DecryptRequest = crate::dapr::dapr::proto::runtime::v1::DecryptRequest;
+
+/// Encryption request options
+type EncryptRequestOptions = crate::dapr::dapr::proto::runtime::v1::EncryptRequestOptions;
+
+/// Decryption request options
+type DecryptRequestOptions = crate::dapr::dapr::proto::runtime::v1::DecryptRequestOptions;
+
+type StreamPayload = crate::dapr::dapr::proto::common::v1::StreamPayload;
```

Comment:
These types should be exposed allowing us to decouple from the protos
```suggestion
/// Encryption gRPC request
pub type EncryptRequest = crate::dapr::dapr::proto::runtime::v1::EncryptRequest;

/// Decrypt gRPC request
pub type DecryptRequest = crate::dapr::dapr::proto::runtime::v1::DecryptRequest;

/// Encryption request options
pub type EncryptRequestOptions = crate::dapr::dapr::proto::runtime::v1::EncryptRequestOptions;

/// Decryption request options
pub type DecryptRequestOptions = crate::dapr::dapr::proto::runtime::v1::DecryptRequestOptions;

type StreamPayload = crate::dapr::dapr::proto::common::v1::StreamPayload;
```